jekyll/_cci2/container-runner.adoc

```diff
@@ -321,6 +321,10 @@ Container runner needs the following Kubernetes permissions:
 ** Get
 ** List
 
+If link:container-runner-installation.adoc#enable-rerun-job-with-ssh[Rerun job with SSH] is enabled, the following permissions are also required:
+* link:https://gateway-api.sigs.k8s.io/api-types/gateway/[Gateways], Services
+** Get
```

Comment:
@christian-stephen just FYI I think this part is unfinished, or maybe is not needed?

Reply:
It's needed. That's how we listed the other existing RBAC permissions.

Reply:
Oh my bad I see now! Just a space needed to get it to render correctly